Dockerfile

```diff
@@ -0,0 +1,10 @@
+# syntax=docker/dockerfile:1.4
+ARG PYTHON_VERSION=2.7
+FROM python:$PYTHON_VERSION
+EXPOSE 8000
+WORKDIR /app 
+COPY requirements.txt /app
+RUN pip install -r requirements.txt --no-cache-dir
+COPY . /app 
+ENTRYPOINT ["python"] 
```

Comment:
I didn't check, but should this be "python2.7"?

Reply:
I didn't check either, but I assume the container image only comes with one python executable and therefore doesn't need the version in the command.